src/api/api.ts

```diff
@@ -19,8 +20,7 @@ const requestSigning = (
   return new Promise((resolve, _reject) => {
     webContents.send(IPC_CHANNELS.API, request);
 
-    ipcMain.on(IPC_CHANNELS.API, function _listener(_event, arg) {
-      const response = arg as Omit<JsonRPCResponse, 'jsonrpc'>;
+    ipcBridgeMain(ipcMain).api.on(function _listener(_event, response) {
```

Comment:
Can we use an arrow function here, and remove the `_`?

Reply:
It has to be a named function as otherwise we can't remove the listener later. But I can remove the `_`